hw3/index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="reset.css">
+	<link rel="stylesheet" type="text/css" href="style-hw3.css">
+	<link rel="stylesheet" type="text/css" href="response.css">
+</head>
+<body>
+	<header class="main-header">
+		<div class="logo">
+			<img src="image/logo.png" class="logo__img"> 
+			<div class="logo__name">Global</div>
+		</div>
+		<nav class="main-nav">
+			<ul class="main-nav__list">
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Home</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Portfolio</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">About</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Services</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Projects</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Contact</a>
+				</li>
+			</ul>
+		</nav>
+	</header>
+	<div class="banner">
```

Comment:
please don't start BEM from container, start it from functional blocks. Like menu, images, and so on